### ISSUE TITLE
Support flow ~~0.82~~ 0.85

### ISSUE THREAD

**Issue by PDarkTemplar** [atlassian/react-beautiful-dnd#851]
In flow 0.82 there is breaking change:
```
Removed the ability to use functions as type annotations. This ability was
originally designed to support ES3-style classes, but in practice causes
confusion and missed errors. We found that the vast majority of added errors
were detecting legitimate bugs, and the remainder could be easily updated to
ES6 classes.
```

And now `react-beautiful-dnd` fail to pass flow check and throw flow error.

**Comment by alexreardon** [comment]
Thanks. We will be updated to flow `0.82` in `version 10`

#838

**Comment by mandx** [comment]
FWIW the only problem I have found with v9.0 and Flow 0.82 is that they changed the type definition for the `React.Node` type, specifically that `undefined` is now disallowed as a return value for `render()` methods (`null` is still allowed). I'm not using a lot of this library so only three places I had to change to make Flow happy, and the change is simply replace `?Node` with `Node` (just removing the `?` makes it OK, because `null` is also a `React.Node`).

**Comment by alexreardon** [comment]
Good news, we will be shipping `0.83` support in version 10

**Comment by alexreardon** [comment]
#838

**Comment by alexreardon** [comment]
I think I have fixed all the issues 💃 
I hope to ship v10 in the next 1-3 weeks 👍

**Comment by alexreardon** [comment]
We will be supporting `0.83`

**Comment by alexreardon** [comment]
Shipped in [10.0](https://github.com/atlassian/react-beautiful-dnd/releases/tag/v10.0.0)
